engine/src/flutter/lib/web_ui/lib/src/engine/semantics/progress_bar.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'label_and_value.dart';
+import 'semantics.dart';
+
+/// Indicates a progress bar element.
+///
+/// Uses aria progressbar role to convey this semantic information to the element.
+///
+/// Screen-readers take advantage of "aria-label" to describe the visual.
+class SementicsProgressBar extends SemanticRole {
+  SementicsProgressBar(SemanticsObject semanticsObject)
+    : super.withBasics(
+        EngineSemanticsRole.progressBar,
+        semanticsObject,
+        preferredLabelRepresentation: LabelRepresentation.ariaLabel,
+      ) {
+    setAriaRole('progressbar');
+
+    // Progress indicators in Flutter use values between 0.0 and 1.0
+    setAttribute('aria-valuemin', "0");
```

Comment:
we may need a a new property for maxValue and minValue to propergate the value range. Hardcoded to 0 and 100 may be too opinionated

Reply:
Add new properties maxValue and minValue.